.github/pull_request_template.md

```diff
@@ -11,6 +11,7 @@ This PR fixes/adds/improves/...
 - [ ] Added implementation
 - [ ] Added / updated tests
 - [ ] Added / updated documentation
+- [ ] Added changelog snippet
```

Comment:
Maybe we want to link to the docs here to explain what is meant?

Reply:
Well, people are supposed to read the contribution docs to find out what that means xD
Also, the link to the docs is in the next line.

Reply:
Fair